bin/generate-api-doc.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const fs = require('fs')
+const map = require('lodash.map')
+const flow = require('lodash.flow')
+const sortBy = require('lodash.sortby')
```

Comment:
Yeah, I'm not really fussed either way. The only real reason for going with individual dependencies is it makes it *slightly* easier to see which parts of lodash we're using, just by looking at `package.json`. This code is only for development, so size isn't really a factor.